components/discord_bot/actions/list-channel-invites/list-channel-invites.mjs

```diff
@@ -0,0 +1,35 @@
+import common from "../common.mjs";
+import constants from "../../constants.mjs";
+
+const { discord } = common.props;
+
+export default {
+  ...common,
+  key: "discord_bot-list-channel-invites",
+  name: "List Channel Invites",
+  description: "Return a list of invitees for the channel. Only usable for guild channels.",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...common.props,
+    channelId: {
+      propDefinition: [
```

Comment:
`channelId` is already included in common.props, so you can remove this `props` section.

Reply:
In this case I had to do it a bit different since the property `notAllowedChannels` in `channelId` property is a bit different